widget/class-widget.php

```diff
@@ -31,7 +31,10 @@ function __construct() {
    * @param array $args     Widget arguments.
    * @param array $instance Saved values from database.
    */
-   public function widget( $args, $instance ) {
+   public function widget( $args, $instance ) {      
+
+      // Declare to use for queries used to grab page links where WIVM plugin shortcodes were added
+      global $wpdb;
```

Comment:
Sorry I keep adding these here since I can't comment on other lines, but can you remove places where you used all caps like "IF" And "ELSE". We try not to use all caps since people can perceive it as yelling. That was kind of what drove me to ask you to make "OR" to "or". Thanks a ton.